dist/index.html

```diff
@@ -3,9 +3,32 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Webpack Exercise</title>
-<script defer src="main.js"></script></head>
+    <title>Leaderboard</title>
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
 <body>
-    <h1>Hello webpack!</h1>
+    <h1>Leaderboard</h1>
+    <div>
+      <section class="recent-score">
+        <div>
+          <h2>Recent scores</h2>
+          <button type="button">Refresh</button>
+        </div>
+        <ul>
+          <li>Name: 100</li>
+          <li>Name: 20</li>
+          <li>Name: 50</li>
+          <li>Name: 78</li>
+          <li>Name: 125</li>
+          <li>Name: 77</li>
+          <li>Name: 42</li>
+        </ul>
+      </section>
+      <section class="add-score">
+        <h2>Add your score</h2>
+        <input type="text" placeholder="Your Name">
+        <input type="text" placeholder="Your score">
+        <button type="button">Submit</button>
+      </section>
```

Comment:
- [ ] Great implementation of webpack, evrything works as expected. The only thing I noticed is that the HTML code does not look like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), styling is not required for this milestone. Just make sure to place content on columns, and align the inputs  😉 

Your markup | Wireframe
------------- | -------------
<img width="432" alt="Screenshot 2022-06-14 at 22 40 53" src="https://user-images.githubusercontent.com/77038610/173664780-ff15f27b-fbdb-47c7-81e2-dac60ed291b2.png">  | <img width="614" alt="Screenshot 2022-06-14 at 22 40 32" src="https://user-images.githubusercontent.com/77038610/173664731-03655fd2-9ffe-4e39-8530-1c6961d5013a.png">